GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -27,6 +27,7 @@
 using Microsoft.VisualBasic;
 using Microsoft.VisualStudio.Threading;
 using ResourceManager;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement;
```

Comment:
Can be removed as well as `using Microsoft.VisualBasic;`
Intellisense has become kind of hyperactive adding `using` statements.

Reply:
🤦‍♂️